src/cmd/starpach/main.go

```diff
@@ -0,0 +1,161 @@
+// Command starpach is a tool for developers of Pachyderm.
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+	"runtime/pprof"
+	"sort"
+	"time"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/cmdutil"
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/log"
+	"github.com/pachyderm/pachyderm/v2/src/internal/pctx"
+	ourstar "github.com/pachyderm/pachyderm/v2/src/internal/starlark"
+	"github.com/spf13/cobra"
+	"go.starlark.net/starlark"
+	"go.uber.org/zap"
+	"golang.org/x/exp/maps"
+)
+
+var (
+	verbose    bool
+	profile    bool
+	gofh, stfh *os.File
```

Comment:
These could be `io.WriteCloser`s. I _think_ that might better document how they are used, but ICBW.